src/mame/drivers/plygonet.cpp

```diff
@@ -76,7 +76,13 @@
 #include "screen.h"
 #include "speaker.h"
 #include "tilemap.h"
+#include "png.h"
+#include "emuopts.h"
+#include "fileio.h"
```

Comment:
Please move png.h down below the rest of the src/emu stuff as it comes from src/lib/utils,

Reply:
Removed all three of the above, as with the debug-drawing code stripped out, the includes aren't necessary at all anymore.